### PR TITLE
“Adding new problem”

### PR DESCRIPTION
**Pull request by khanz00024**
“For assignment 3 of SF-1"

### CODE REVIEW

File: roll-numbers/description.md
Comment (DavidHack07) [inline roll-numbers/description.md:1]: The story background element helped me understand the question better, the examples make sense.

**Comment by khanz00024** [comment]
thank you

File: roll-numbers/description.md
Comment (DavidHack07) [inline roll-numbers/description.md:1]: You could also include how to handle edge cases such as if the list contains numbers or special characters.

File: roll-numbers/io.json
Comment (DavidHack07) [inline roll-numbers/io.json:1]: Including what output there should be for inputs with special characters and numbers would be good as well.

Comment (khanz00024) [reply roll-numbers/io.json:1]: ok...i am making changes...

**Pull request merged**